Makefile

```diff
@@ -118,6 +124,7 @@ dev-infra-up: compose.yaml nkey  ## Starts local services to simplify local deve
 			done
 
 dev-infra-down: compose.yaml  ## Stops local services used for local development.
+	@rm -f .audit/audit.log
```

Comment:
Would we want to be able to view this file if we take down the infra? I think this should probably be in the `clean` target.